src/view/View.js

```diff
@@ -1271,11 +1271,12 @@ new function() { // Injection scope for event handling on the browser
                     point, prevPoint)
             // Next handle the hit-item, if it's different from the drag-item
             // and not a descendant of it (in which case it would already have
-            // received an event in the call above).
+            // received an event in the call above). Translate mousedrag to
+            // mousemove, since drag is handled above.
             || hitItem && hitItem !== dragItem
                 && !hitItem.isDescendant(dragItem)
-                && emitMouseEvent(hitItem, null, type, event, point, prevPoint,
-                    dragItem)
+                && emitMouseEvent(hitItem, null, type === 'mousedrag' ?
+                'mousemove' : type, event, point, prevPoint, dragItem)
```

Comment:
@sasensi this line should be indented by 4 more spaces, as it's inside a method call.

Reply:
@sasensi could you address this?

Reply:
Ok, done :)